app/models/manageiq/providers/lenovo/physical_infra_manager/physical_server.rb

```diff
@@ -3,5 +3,8 @@ class Lenovo::PhysicalInfraManager::PhysicalServer < ::PhysicalServer
     def name
       "physical_server"
     end
+
+    belongs_to :ext_management_system, :foreign_key => "ems_id", :class_name => "ManageIQ::Providers::Lenovo::PhysicalInfraManager"
+    has_one :host, :foreign_key => "service_tag", :primary_key => "serial_number"
```

Comment:
We don't need the `belongs_to` and `has_one` relationships here as they're defined in the super class `Physical Server`